util/cron/common.bash

```diff
@@ -18,6 +18,8 @@ if [ -z "${OFFICIAL_SYSTEM_LLVM}" ] ; then
     source /data/cf/chapel/setup_system_llvm.bash
   elif [ -f /cray/css/users/chapelu/setup_system_llvm.bash ] ; then
     source /cray/css/users/chapelu/setup_system_llvm.bash
+  elif [ -f /cy/users/chapelu/setup_system_llvm.bash ] ; then
+    source /cy/users/chapelu/setup_system_llvm.bash
```

Comment:
Just a couple lines below there is also a cmake line here that probably needs the same treatment

Reply:
Could we use a COMMON_DIR variable here too in order to simplify things? And so Jenkins configs or people outside of HPE could override COMMON_DIR and supply the appropriate scripts?

Reply:
I'm just trying to get things working again, and then creating followup issues about how to improve things going forward. cmake is not needed for these configs at the moment and I don't care about these scripts (and probably anything on `util/cron`) working outside HPE currently.